code/obj/item/rcd.dm

```diff
@@ -133,7 +133,7 @@ TYPEINFO(/obj/item/rcd)
 	var/list/datum/contextAction/contexts = list()
 
 	get_desc()
-		. += "<br>It holds [matter]/[max_matter] [istype(src, /obj/item/rcd/material) ? material_name : "matter"]  units. It is currently set to "
+		. += "<br>It holds [matter]/[max_matter] [istype(src, /obj/item/rcd/material) ? material_type : "matter"]  units. It is currently set to "
```

Comment:
this one should probably be material name

Reply:
![image](https://github.com/goonstation/goonstation/assets/51260013/413bcf08-273c-438e-b675-55d7e6d23106)
you have a point